dashboard/config/locales/en.yml

```diff
@@ -211,7 +211,7 @@ en:
     login: 'Sign in'
     invalid_login: 'Invalid login, please try again'
     pair_programming: 'I have a partner at my computer'
-    student_privacy: "Learn more about why you're not seeing your full name <a href='%{student_privacy_blog}' target='_blank'>here</a>."
+    student_privacy_markdown: "Learn more about why you're not seeing your full name [here](%{student_privacy_blog})."
```

Comment:
This change means that the link will no longer be opened in a new tab right?

Reply:
yep! Which is fine; there's no state on this page for the user to lose by navigating away